app/controllers/api/collections_controller.rb

```diff
@@ -0,0 +1,35 @@
+class Api::CollectionsController < Api::BaseController
+  include ManageMetadataHelper
+  include PiqToDraft
+
+  def create
+    message = 'Request params are not a valid collection draft.'
+    status = :unprocessable_entity
+    errors = []
+
+    if PiqToDraft.valid?(params)
+      user = User.where(urs_uid: 'piq_user').last || User.create({
+        urs_uid: 'piq_user',
+        provider_id: 'NASA_MAAP'
+      })
+      user.save!
+
+      native_id = params[:collection_info][:short_title]
+      draft = CollectionDraft.find_or_create_by(native_id: native_id)
+      draft.draft = {}
+      draft.provider_id = 'NASA_MAAP'
```

Comment:
Will the provider for drafts from the API never be `ESA_MAAP`?

Reply:
good question, we should make this configurable, right now there is no field in the questionnaire for provider, I think it's ok as NASA_MAAP for now since this workflow is mostly intended for users of the NASA MAAP but I have added this card: https://trello.com/c/tF92vx7x